fixd.js

```diff
@@ -1,14 +1,30 @@
-const fs = require('fs')
-const path = require('path')
-const fixd = require('./src/index')
+'use strict'
+const merge = require('lodash.merge')
 
-// Auto-load built-in plugins
-try {
-  fs.readdirSync(path.resolve(__dirname, './src/plugins')).forEach((f) => {
-    fixd.use(f.replace('.js', ''))
-  })
-} catch (e) {
-  throw new Error('Error loading built-in plugins', e.message)
+const fixd = {
+  /**
+   * Add a fixture to the fixd library and freezes if not in namespace
+   * @param {String} name A string name to represent the fixture
+   * @param {*} value The fixture value
+   */
+  add: (name, val) => {
+    if (Object.keys(fixd).indexOf(name) >= 0) {
+      throw new Error(`Cannot add fixture to already reserved namespace '${name}'.`)
+    }
+    fixd[name] = Object.freeze(val)
```

Comment:
Note this will only work with objects and arrays.  If we want to support primitives such as strings and numbers, we'll have to have some kind of object wrapper cache internally.

FWIW, I'm fine with an object/array for the simplicity and initial release.  The user can just format their string as an object if needed.